crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -955,12 +971,23 @@ impl<'db> TypeInferenceBuilder<'db> {
         // `follow_nonexistent_import_bare_to_module()`.
         let ast::StmtImportFrom { module, level, .. } = import_from;
         tracing::trace!("Resolving imported object {alias:?} from statement {import_from:?}");
+        let module = module.as_deref();
         let module_name = if let Some(level) = NonZeroU32::new(*level) {
-            self.relative_module_name(module.as_deref(), level)
+            tracing::debug!(
+                "Resolving imported object '{}' from module '{}' relative to file '{}'",
```

Comment:
Same here. We should avoid making debug too verbose. Maybe consider making it a trace or removing it entirely

Reply:
I made it a trace